spyder/widgets/sourcecode/tests/test_indentation.py

```diff
@@ -84,5 +84,35 @@ def test_selection_indent(code_editor_indent_bot):
                     )
 
 
+def test_fix_indentation(code_editor_indent_bot):
+    """Test fix_indentation() method."""
+    ed, qtbot = code_editor_indent_bot
```

Comment:
Minor nitpick, but maybe editor is a better word :-), we are not saving that much space bellow anyway.